coalib/coala_modes.py

```diff
@@ -26,7 +26,8 @@ def mode_normal(console_printer, log_printer, args, debug=False):
         console_printer=console_printer,
         args=args,
         debug=debug)
-
+    if log_printer:
+        logging.warn('Deprecated')
```

Comment:
add 1 space here before the `logging.warn` @arpitsimt

Reply:
Shouldn't it be `logging.warning('Warning: The use of log_printer is deprecated')`?